Automattic/jetpack
# Fix/dotcom default settings for newsletter byline

This PR make sure that the default value are as expected for the byline values. 

We treat these value as true if they are not set yet. This PR fixes it so that the endpoint also returns the value as true. 

This should only have an affect on simple sites since the UI in Jetpack already uses the correct default values. 

## Proposed changes:

* Make the default value of `jetpack_gravatar_in_email`,  `jetpack_author_in_email` and `jetpack_post_date_in_email` to be set as true. 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
Load this PR Do not make any changes to the settings. Make sure that they are empty. 
Notice that now we return the default value as true. 
visit https://developer.wordpress.com/docs/api/console/ 

Make a request to the `/sites/$site/setting` endpoint.  See 
<img width="564" alt="Screenshot 2024-07-19 at 12 44 34 PM" src="https://github.com/user-attachments/assets/be317c79-b3ea-4c34-8d84-d45bbbf426f4">
